Add announcement blog for public sharing (#39)
Contenda-Team/Contenda-landing-page: src/posts/sharepear.md

---
layout: "../layouts/BlogPost.astro"
title:
    "Announcing Public Sharing: Share Your Drafts with the World on Contenda!"
slug: public-sharing
description:
    "You can now publicly share your blog drafts on Contenda with just one
    click!"
author: "Cassidy Williams"
added: "Apr 18 2023"
tags: ["announcements"]
---

Calling all content creators, developer advocates, and storytelling savants!

We're thrilled to announce that Contenda now allows you to share your generated
blog posts publicly with the world! Previously, you would generate your content
and it would be tied exclusively to your account (which is still true), but now
you have the ability to share posts publicly with your teams, editors, friends,
and enemies.

## How does it work?

We've got a whooole separate blog about how the feature was built! But, outside
of that, what you need to know:

> 1. You create content with Contenda, and you edit it to your heart's content

Reviewer: 1. You create content and edit with Contenda

Reviewer: (Because you say heart's content for bullet point 4)

> 1. You create content with Contenda, and you edit it to your heart's content

Author: ```suggestion
1. You create content and edit it on the Contenda platform, all willy-nilly
```
